### PR TITLE
Add nanmedian to numpy: frontend

### PR DESCRIPTION
**Pull request by aryanxk02**
Fixes https://github.com/unifyai/ivy/issues/9154

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py [hunk 1]
```diff
@@ -302,3 +302,55 @@ def test_numpy_cov(
         dtype=dtype[0],
         test_values=False,
     )
+
+
+# nanmedian
+@handle_frontend_test(
+    fn_tree="numpy.nanmedian",
+    dtype_and_a=statistical_dtype_values(function="nanmedian"),
```
Comment (RickSanchezStoic) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py:310]: make `function=mean` and see if it works

Comment (aryanxk02) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py:310]: Okay

**Comment by aryanxk02** [comment]
@RickSanchezStoic still the same error.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**